libpod/adapter/runtime_remote.go

```diff
@@ -452,3 +454,29 @@ func (r *LocalRuntime) RemoveVolumes(ctx context.Context, c *cliconfig.VolumeRmV
 	}
 	return iopodman.VolumeRemove().Call(r.Conn, rmOpts)
 }
+
+func (r *LocalRuntime) Push(ctx context.Context, srcName, destination, manifestMIMEType, authfile, signaturePolicyPath string, writer io.Writer, forceCompress bool, signingOptions image.SigningOptions, dockerRegistryOptions *image.DockerRegistryOptions, additionalDockerArchiveTags []reference.NamedTagged) error {
+
+	tls := true
+	if dockerRegistryOptions.DockerInsecureSkipTLSVerify == types.OptionalBoolTrue {
+		tls = false
+	}
+	reply, err := iopodman.PushImage().Send(r.Conn, varlink.More, srcName, destination, tls, signaturePolicyPath, "", dockerRegistryOptions.DockerCertPath, forceCompress, manifestMIMEType, signingOptions.RemoveSignatures, signingOptions.SignBy)
```

Comment:
Not for this PR, but something to consider.  This is a humongous function signature.  Might be good to create a struct with a number of these within it and pass that instead.  Food for thought.